dev-libs/openssl/openssl-1.0.2f.recipe

```diff
@@ -40,6 +40,11 @@ REQUIRES="
 	lib:libz$secondaryArchSuffix >= 1.2.3
 	ca_root_certificates
 	"
+if [ -n "$secondaryArchSuffix" ]; then
+	REQUIRES="$REQUIRES
+		openssl == $portVersion base
+		"
+fi
```

Comment:
Could you explain this? I think Adrien removed this as he thought the requirement could be left out.